src/codegen/strength_reduce/expression_values.rs

```diff
@@ -472,7 +472,23 @@ fn multiply_values(
         })
         .collect()
 }
-
+fn get_value(
+    _signed : bool,
+    _l :&Value,
+    _r :&Value,
```

Comment:
please don't start variable names with `_`. Also `left` and `right` is better than `l` and `r`.

Reply:
In addition, please, rename `_signed` to `signed`. In Rust, parameters that start with an underscore are not used in the function.